### PR TITLE
added prometheus endpoint, added Dockerfile

### PR DESCRIPTION
**Pull request by rpurdel**

### CODE REVIEW

**Review by mihhu — APPROVED** [review]

**Comment by saghul** [comment]
Please drop any internal references: https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-contributing#for-8x8-employees

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:12-alpine
```
Comment (saghul) [inline Dockerfile:1]: Node 12 is EOL, please use the latest LTS version.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /excalidraw-backend
+
+COPY package.json package-lock.json ./
```
Comment (saghul) [inline Dockerfile:5]: Please copy all the files instead of doing it in stages and thus having multiple layers.

File: package.json [hunk 3]
```diff
@@ -42,7 +42,8 @@
         "express": "4.17.1",
         "socket.io": "^2.5.0",
         "ts-node-dev": "^1.1.8",
-        "typescript": "4.2.3"
+        "typescript": "4.2.3",
+        "socket.io-prometheus-metrics": "1.0.6"
```
Comment (saghul) [inline package.json:46]: Did you add this manually? npm keeps deps sorted alphabetically.

File: src/index.ts [hunk 4]
```diff
@@ -41,6 +42,11 @@ const io = socketIO(server, {
   },
 });
 
+// exposes host:9090/metrics for prometheus
```
Comment (saghul) [inline src/index.ts:45]: You haven't exposed this port in the Dockerfile, should it be there?

**Review by saghul — CHANGES_REQUESTED** [review]
Generally speaking, please use different commits for different features / fixes. The prometheus endpoint is not related to the Dockerfile, for example.

Also selft some other comments, PTAL.

**Pull request closed without merging**